src/pages/Lecture/LectureFooter.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function LectureFooter() {
+  return (
+    <StyledDiv>
+      <div>
+        <FontAwesome>
+          <i class="fas fa-chevron-left" />
+          이전 강의
+        </FontAwesome>
+        <FontAwesome margin="0 64px" padding="6px 0">
+          <i class="fas fa-check" />
```

Comment:
```suggestion
          <i className="fas fa-check" />
```
jsx에서는 class 사용 안합니다